src/main/java/org/osiam/resources/scim/Resource.java

```diff
@@ -41,27 +40,16 @@
     private final String id;
     private final String externalId;
     private final Meta meta;
-    @JsonProperty(required = true)
     private final Set<String> schemas;
 
-    protected Resource(@JsonProperty("id") String id,
-                       @JsonProperty("externalId") String externalId,
-                       @JsonProperty("meta") Meta meta,
-                       @JsonProperty(value = "schemas", required = true) Set<String> schemas) {
+    protected Resource(String id, String externalId, Meta meta, Set<String> schemas) {
         this.id = id;
         this.externalId = externalId;
         this.meta = meta;
         if (schemas == null || schemas.isEmpty()) {
             throw new SCIMDataValidationException("Schemas cannot be null or empty!");
         }
-        this.schemas = schemas;
-    }
-
-    protected Resource(Builder builder) {
```

Comment:
The builder-based constructor of `Resource` is gone anyway. There's simply no client, who could call it anymore.